crates/bevy_animation_rig/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+use bevy_app::{AppBuilder, CoreStage, Plugin, StartupStage};
+use bevy_asset::AddAsset;
+use bevy_ecs::{
+    schedule::{ParallelSystemDescriptorCoercion, SystemLabel},
+    system::IntoSystem,
+};
+
+mod skinned_mesh;
+use bevy_transform::TransformSystem;
```

Comment:
Can you move this to be grouped with the other `use bevy_` declarations at the top?